TCAT/Cells/LargeDetailTableViewCell.swift

```diff
@@ -40,7 +40,7 @@ class LargeDetailTableViewCell: UITableViewCell {
         titleLabel.font = UIFont.systemFont(ofSize: UIFont.systemFontSize)
         titleLabel.lineBreakMode = .byWordWrapping
         titleLabel.textColor = .primaryTextColor
-        titleLabel.text = "Board"
+        titleLabel.text = direction != nil && direction.type == .transfer ? "Bus becomes" : "Board"
```

Comment:
jw when would a direction be nil? i assume every cell would be associated with a direction

Reply:
Yeah it's a bit weird but this stuff is accessed on init before Direction is called